index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MyPortfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap">
+    <script src="https://kit.fontawesome.com/366c518b05.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header>
+        <nav>
+            <a href="#" class="firstTitle"> </a>
+            <a href="#" class="hamburgerButton"><i class="fa-solid fa-bars"></i></a>
+        </nav>
+    </header>
+    <section class="heroIntro">
+        <div class="heroIntroContainer">
+            <h1 class="heroIntroHeading">
+                Hey there. I'm
+            </h1>
+            <h1 class="secondHeroIntroHeading">
```

Comment:
- [ ] [OPTIONAL] There should be only one `h1` in an HTML file because of semantics and it helps search engines and web crawlers easily understand the main subject of your page.  You can make use of `CSS` to separate your texts on different lines by adding a display of `block`